src/extension/content-script/webbtc.js

```diff
@@ -24,6 +25,16 @@ async function init() {
     return;
   }
 
+  browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
+    // forward account changed messaged to inpage script
+    if (request.action === "accountChanged") {
```

Comment:
```suggestion
    if (request.action === "accountChanged" && isEnabled) {
```

Reply:
Make sure it is this way for all providers